src/core/thread.d

```diff
@@ -4786,13 +4786,14 @@ private:
             pstack -= reserve;
             *(cast(EXCEPTION_REGISTRATION*)pstack) =
                 EXCEPTION_REGISTRATION( sehChainEnd, finalHandler );
+            auto pChainEnd = pstack;
```

Comment:
`const` ?

Reply:
Not really the style of this file.

Reply:
Normally I'd disagree that that is a good reason not to do it, but the whole file is about to get a re-architect anyway, so it doesn't really matter.

Reply:
Fair enough